cmd/minishift/cmd/config/config.go

```diff
@@ -97,6 +97,30 @@ var (
 	HostFoldersAutoMount = createFlag("hostfolders-automount", SetBool, nil, nil, true)
 
 	ImageCaching = createFlag("image-caching", SetBool, nil, nil, true)
+
+	// Preflight checks (before start)
+	SkipCheckKVMDriver    = createFlag("skip-check-kvm-driver", SetBool, nil, nil, true)
+	WarnCheckKVMDriver    = createFlag("warn-check-kvm-driver", SetBool, nil, nil, true)
```

Comment:
I am missing some doc updates which covers the pre-flight checks and the fact they are configurable and how to do it.